Makefile

```diff
@@ -89,6 +92,10 @@ clean: ## Deletes all build artifacts
 	rm -rf tools.timestamp
 	rm -rf $(BUILD_DIR)
 
+.PHONY: validate_commits
+validate_commits: tools ## Validates git commit messages
+	git-validation -q -run short-subject,message_regexp='^Issue #[0-9]+  ?[A-Z]+.*' -range $(START_COMMIT_MESSAGE_VALIDATION)...
```

Comment:
Are you saying there can be 1 or 2 spaces between the number and text? Why 1 or 2?

Reply:
Well, actually no, but I need a start-commit and the previous commit had two spaces. The plan was to change this in a later commit. But I have a better idea now. I will add a commit before this one updating the README. Then this commit can serve as starting point.

Reply:
Trying to put some safeguards in place for the time to come. ;-)

Reply:
I completely understand the reasoning, just the 2 spaces confused me:D